CHANGELOG.md

```diff
@@ -3,14 +3,13 @@
 New features:
 
 - [#501 Add default session data](https://github.com/alphagov/govuk_prototype_kit/pull/501)
+- [#502 Add Cookies and Privacy policy text](https://github.com/alphagov/govuk_prototype_kit/pull/502)
```

Comment:
this doesn't look like it belongs in this PR

Reply:
I'm cleaning up the fact there's two 'new features' sections in the unreleased section...

Reply:
oh right its moved not an addition, ok